src/Plugin/CKEditorPlugin/DrupalEntity.php

```diff
@@ -8,7 +8,6 @@
 namespace Drupal\entity_embed\Plugin\CKEditorPlugin;
 
 use Drupal\editor\Entity\Editor;
-use Drupal\embed\EmbedButtonInterface;
 use Drupal\embed\EmbedCKEditorPluginBase;
```

Comment:
The DrupalEntityButton plugin, which extends this one, is intended for CKEditor -- so it does need to extend EmbedCKEditorPluginBase. It's not the cleanest, but short of converting EmbedCKEditorPluginBase to a trait I'm not sure how else to approach that...